.github/workflows/deploy-portal-prod.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy a portal to production
+
+on:
+  workflow_dispatch:
+    inputs:
+      portal-name:
+        type: choice
+        description: Which portal to deploy
+        options: 
+        - adknowledgportal
+        - challengeportal
+      tag:
+        required: true
+        type: string
+        description: Which tag to deploy from
```

Comment:
I think I like this approach (manually create a tag, then pick it to deploy) so we can roll back prod to a previous version if needed

Reply:
We use that pattern in IT, thought it fit here too...